src/indicadores_nbi.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+import seaborn as sns
+import pyeph 
+
+# Diccionario de datos 
+años = (2021, 2022)
+houses = {}
+individuals = {}
+for a in años:
+    for i in range(1,5):
```

Comment:
Aca en vez de usar `i` podemos usar `periodo`, así queda claro semanticamente.

Reply:
También podrías asignar `range(1,5)` a una variable  `periodos_bieniales` (puse bienial porque entiendo que los 5 periodos suceden dentro del plazo de 2 años pero ahí podes corregirme aver si entendí bien)

Mejorar este naming está bueno porque otro (en este caso yo) después lee directamente :
`for periodo in periodos_bieniales` y entonces entiende de toque qué estas haciendo.